src/get_clades_to_model.py

```diff
@@ -21,14 +21,17 @@
 # /// script
 # requires-python = ">=3.11"
 # dependencies = [
-#   "virus_clade_utils@git+https://github.com/reichlab/virus-clade-utils/",
+#   "virus_clade_utils@git+https://github.com/reichlab/virus-clade-utils@bsweger/get_nextstrain_ncov_metadata",
```

Comment:
This is the reason the PR is in draft status...until https://github.com/reichlab/virus-clade-utils/pull/26 is merged, this file is instructing `uv` to install the `virus_clade_utils` feature branch that produces the needed ncov metadata.